azclishell/configuration.py

```diff
@@ -90,6 +92,8 @@ def load(self, path):
     def firsttime(self):
         """ sets it as already done"""
         self.config.set('DEFAULT', 'firsttime', 'no')
+        set_global_config_value('core', 'collect_telemetry', ask_user_for_telemetry())
```

Comment:
Should check if the value is already set to true. If it is, you don't need to ask again as we take the CLI value.
If it's not, then ask.

But on first run, if it is already enabled, show a message that we track telemetry. The same message the CLI shows.